libobs/obs.c

```diff
@@ -1935,3 +1937,13 @@ void obs_get_audio_monitoring_device(const char **name, const char **id)
 	if (id)
 		*id = obs->audio.monitoring_device_id;
 }
+
+void obs_set_replay_buffer_filename(char* filename)
+{
+	replay_buffer_filename = filename;
+}
+
+char* obs_get_replay_buffer_filename()
```

Comment:
The asterisk should immediately precede the function name.
"char* obs_get_replay_buffer_filename" should be "char *obs_get_replay_buffer_filename"